pkg/apis/serving/v1/route_lifecycle_test.go

```diff
@@ -428,54 +428,95 @@ func TestRouteNotOwnedStuff(t *testing.T) {
 	apistest.CheckConditionFailed(r, RouteConditionReady, t)
 }
 
-func TestCertificateReady(t *testing.T) {
-	r := &RouteStatus{}
-	r.InitializeConditions()
-	r.MarkCertificateReady("cert")
-
-	apistest.CheckConditionSucceeded(r, RouteConditionCertificateProvisioned, t)
-}
+func TestCertificateProvision(t *testing.T) {
+	message := "CommonName Too Long"
 
-func TestCertificateNotReady(t *testing.T) {
-	r := &RouteStatus{}
-	r.InitializeConditions()
-	r.MarkCertificateNotReady(&netv1alpha1.Certificate{})
-
-	apistest.CheckConditionOngoing(r, RouteConditionCertificateProvisioned, t)
-}
-
-func TestCertificateNotReadyWithBubbledUpMessage(t *testing.T) {
-	cert := &netv1alpha1.Certificate{
-		Status: netv1alpha1.CertificateStatus{
-			Status: duckv1.Status{
-				Conditions: duckv1.Conditions{
-					{
-						Type:   "Ready",
-						Status: "False",
-						Reason: "CommonName Too Long",
+	cases := []struct {
+		name   string
+		cert   *netv1alpha1.Certificate
+		status corev1.ConditionStatus
+
+		wantMessage string
+	}{
+		{
+			name:        "Ready with empty message",
+			cert:        &netv1alpha1.Certificate{},
+			status:      corev1.ConditionTrue,
+			wantMessage: "",
+		},
```

Comment:
Do you mind compacting this to match the list style we use in other parts of the repo

see: https://github.com/knative/serving/blob/315a94c673ea9133875406e695edd50b46a04776/pkg/apis/serving/v1/route_lifecycle_test.go#L78-L83

Reply:
Sure, thanks for your suggestion. will do this change.